aether-ui/aether/ui/assets/conf/webpack.prod.js

```diff
@@ -40,5 +90,11 @@ module.exports = buildConfig({
   production: true,
   stylesAsCss: true,
   jsFilename: `${prefix}-${revision || '[hash]'}.js`,
-  cssFilename: `${prefix}-${revision || '[chunkhash]'}.css`
+  cssFilename: `${prefix}-${revision || '[chunkhash]'}.css`,
+  output: process.env.CDN_PLATFORM ? {
+    // Tell django to use this URL to load packages
+    // and not use STATIC_URL + bundle_name
+    publicPath: process.env.WEBPACK_URL
+  } : {},
+  plugins: getCDNPlugin()
```

Comment:
Actually, do we need this?

This plugin is executed when we push the image to docker hub, not when we start the app. Do we have the final credentials for each instance in that step? Not. :disappointed:

Reply:
That is true. I assume we will have to have a docker-hub image for `ui-assets` to rebuild webpack static files  to post to a cloud service from a setup like `bootstrap` or can we build from within the `ui` image?

Reply:
Because this is only our deployment problem, I think that we could split the release step in two, one for the release itself (push to dockerhub) and the other one to push to GCS our "commit hash" and the static content.

I mean move this, https://github.com/eHealthAfrica/aether/blob/develop/scripts/release.sh#L64-L82, to another bash file and create another entry in travis for it and the static content.